src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ImageComponent.tsx

```diff
@@ -38,13 +39,11 @@ export function ImageComponent(props: IImageProps) {
     useSelector((state: IRuntimeState) => state.profile.profile.profileSettingPreference.language);
   const width = props.image.width || '100%';
   const align = props.image.align || 'center';
-  const altText = props.getTextResourceAsString(props.textResourceBindings.altTextImg) || 'image';
+  const altText = props.getTextResourceAsString(props.textResourceBindings.altTextImg);
 
-  let imgSrc = props.image.src.nb;
-  if (language === 'en' && props.image.src.en) {
-    imgSrc = props.image.src.en;
-  } else if (language === 'nn' && props.image.src.nn) {
-    imgSrc = props.image.src.nn;
+  let imgSrc = props.image.src[language] || props.image.src[Object.keys(props.image.src)[0]];
```

Comment:
This is IMO a better solution than the current, but I'm not sure we should change the default fallback to be the first entry, as it would be considered a breaking change. 

If we do this, we also need to update the docs accordingly which states that `nb` value would be used if the language is not specified.

@lorang92 @mjulstein thoughts?

Reply:
As we try to keep most docs and communications in English in the hope that it will be used internationally, we can't really have nb as the default.

Reply:
I dont disagree :) Just worried about the breaking change... How do we typically deal with that @lorang92 ?

If we change this, we should probalby also not require `nb` to be used (line 24 in `ImageComponent.tsx`)

Reply:
Line 24 is not a requirement but a typescript assertion (with no runtime impact). The declaration of `nb` as not nullable is wrong anyway, because a missing null check on nb is something we would want typescript to draw attention to.

Reply:
Its not a runtime requirement, but TS will complain if its not set
![image](https://user-images.githubusercontent.com/2082481/141739175-94a1ff78-1dd7-4e07-9f54-253c5052f42f.png)

In this PR you are also removing `nb` as the default, which makes this requirement not appliccable anymore. If we include this PR, `nb` value should also be optional.

Reply:
It won't complain when we take user data from the json file and cast it. I agree this instance is wrong but all declarations in the whole project where we take user data and cast to a type with non nullable fields are equally wrong.

None of these properties are used anymore (after this pr), so in practice `nb`, `nn` and `en` are just to make the code easier to read.

Reply:
> I agree this instance is wrong

Then why the reluctance to fix it? :) 

There are probably lots of other things that are incorrect here and there. If we dont fix them when we spot them, we'll never get into a better situation.

To be clear, I'm only saying that `nb` value should be equally optional as `nn` and `en` in the interface;

```typescript
export interface IImageProps {
  id: string;
  nb?: string; // <-- added ?.
  nn?: string;
  en?: string;
  [language: string]:string;
}
```

Reply:
I'm reluctant because you first said that I should remove all the extra declarations now that I added the `[language: string]: string;` part, and then changed your mind. It is not hard to visualise this change while we wait for others to look at this, so this is not something i "HAVE TO DO RIGHT NOW".

Then we ended up discussing related topics about how typescript works.

Reply:
Sorry, I gave a "thumbs up" on your comment about keeping those values for intellisense hints. I could have been more clear about that being a good idea. I dont have any objections to keep them for that reason.

I didnt say you have to fix it immeditately - I just meant that in the current PR state, this needs to be fixed before its merged. Its of course fine to wait for more feedback from others :)

Reply:
@lorang92 Just `nb` as fallback (or error?), or first `nb` then just pick whatever is first?

Reply:
@ivarne that's a good question. In the other parts of our solution, we only return "nb" or otherwise nothing if the fallback actually does not exist. I guess the first alternative is most fitting with this way of handling it.

Reply:
Okay, so we should keep nb as default/fallback.

@ivarne Is it clear to you what changes are needed?

Reply:
Nope, not entierly, which is why this has been forgotten. Thanks for bringing it up. As I understand its either 
```suggestion
  let imgSrc = props.image.src[language] || props.image.src.nb;
```
or
```suggestion
  let imgSrc = props.image.src[language] || props.image.src.nb || props.image.src[Object.keys(props.image.src)[0]];
```

Reply:
And obviously make write `nb?` in the type declaration when the important discussion has been settled.

Reply:
I think this suggestion makes sense `let imgSrc = props.image.src[language] || props.image.src.nb;` , and thus keep `nb` required, and used as a fallback language. This way its aligned with how the rest of the application works.

Moving away from nb as default/fallback language would be a separate task.

WDYT @lorang92 ?

Reply:
Sounds great! 👌